supervised/loglizer/loglizer/models/LogClustering.py

```
@@ -0,0 +1,137 @@
+Authors: 
+    LogPAI Team
+
+Reference: 
+    [1] Qingwei Lin, Hongyu Zhang, Jian-Guang Lou, Yu Zhang, Xuewei Chen. Log Clustering 
+        based Problem Identification for Online Service Systems. International Conference
+        on Software Engineering (ICSE), 2016.
+
+"""
+
+import numpy as np
+import pprint
```

Ruff (0.8.2)

`pprint` imported but unused

Remove unused import: `pprint`

(F401)